### DISCUSSION TITLE
Unable to be Student Verification

### DISCUSSION THREAD

**Discussion by AzmainFayeqBhuiyan** [community/community#59347] [GitHub Education]
### Hi everyone,

Hey AzmainFayeqBhuiyan,Unfortunately, we were unable to verify your academic affiliation based on the information you submitted on Jun 24, 2023 for @AzmainFayeqBhuiyan.Common verification problems and suggested solutions are documented here: https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/why-wasnt-my-application-to-global-campus-for-students-approved.Once you have resolved the issues with your sumbission, visit us again and reapply.Have questions? Join the GitHub Education Community for answers. The GitHub Education Team

I have tried to verify countless times, added my full name to my Bio and "My Name" in the Github Profile, have added my school email (alongside my regular gmail), added my school's website link to in the Profile area also. Nothing works...

I have also tried to do this from my phone yet it does not work.

If anyone has any help/solutions, please help me. Thanks :D

**Comment by samus-aran** [comment]
Hi @AzmainFayeqBhuiyan 👋 

Apologies for the delayed response from the team, we hope you were able to resolve your rejection reasons and receive a successful verdict on your application. I'll be closing this discussion due to lack of recent activity. If you still haven't received your Global Campus benefits please feel free to re-open and we'll be happy to assist!
